physics/docs/pdftxt/RRFS_v1alpha_suite.txt

```diff
@@ -24,7 +24,7 @@ The RRFS_v1alpha physics suite uses the parameterizations in the following order
 \code
 <?xml version="1.0" encoding="UTF-8"?>
 
-<suite name="FV3_RRFS_v1alpha" lib="ccppphys" ver="5">
```

Comment:
Please replace `lib="ccppphys" ver="1"` with just `version="1"` to be consistent with the new syntax.